frontend/app/src/util/AppNavigation.ts

```diff
@@ -159,7 +159,7 @@ export class StrategyV1 {
       this.appPages.find(appPage =>
         // The page name is embedded at the end of the URL path, and if not, we are in the main page.
         // See https://github.com/streamlit/streamlit/blob/1.19.0/frontend/src/App.tsx#L740
```

Comment:
Nit: We should probably update the comment (on StrategyV1 & StrategyV2) to reference the `urlPathname` instead of the `page name`